ansible/system-update.yml

```diff
@@ -5,4 +5,3 @@
   roles:
     - role: system-base-update
     - role: system-package-update
-
```

Comment:
I'll leave it to others to decide on the use and naming of this playbook (`all` implies it's for production systems too).